testing/v8_integration_test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Looks like there is a lot of duplication in this file to just pass "-run_only_v8_canary_test" flag to the last command. Is there no way to change the existing *.sh file to, say, pass "$*" to the last command and configure the Kokoro run in the *.cfg file to pass "-run_only_v8_canary_test" flag?

Reply:
This requires internal changes as well. Can I merge this to fix failing continuous tests, then update internal configs? (Should still be done today)